app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,31 @@
+class CommentsController < ApplicationController
+  def new
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    comment = Comment.new
+    respond_to do |format|
+      format.html { render :new, locals: { comment: } }
+    end
+  end
+
+  def create
+    @user = current_user
+    puts params
+    @post = Post.find(params[:post_id])
+    comment = Comment.new(params.require(:comment).permit(:text))
```

Comment:
### **OPTIONAL**

- [ ] Please enhance your code with the [Strong Parameter](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides.)